### PR TITLE
install the ebs-csi-driver addon when creating aws eks cluster

### PR DESCRIPTION
**Pull request by xjiaqing**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
Update example eksctl file to install the ebs-csi-driver addon when creating aws eks cluster

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.5 (TiDB Operator 1.5 versions)
- [ ] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is relate to: https://github.com/pingcap/docs-tidb-operator/issues/2483
- Other reference link(s): [Amazon EBS CSI driver is now generally available in Amazon EKS add-ons](https://aws.amazon.com/cn/blogs/containers/amazon-ebs-csi-driver-is-now-generally-available-in-amazon-eks-add-ons/)

### CODE REVIEW

File: zh/deploy-on-aws-eks.md [hunk 1]
```diff
@@ -118,7 +136,9 @@ nodeGroups:
       dedicated: pd
     taints:
       dedicated: pd:NoSchedule
-
+    iam:
+      withAddonPolicies:
+        ebs: true
```
Comment (csuzhangxc) [inline zh/deploy-on-aws-eks.md:141]: 后面其他 NodeGroup 示例也加一下这个 ebs policies 吧

Comment (xjiaqing) [reply zh/deploy-on-aws-eks.md:141]: 我看了一下，PR 在示例里面的所有 node group 里面都加了这个权限。

Comment (csuzhangxc) [reply zh/deploy-on-aws-eks.md:141]: 比如 https://github.com/pingcap/docs-tidb-operator/blob/c5877fdc9a3cf875de9557bee5acf53eee9b1828/zh/deploy-on-aws-eks.md?plain=1#L576 这里需要加吗

File: en/deploy-on-aws-eks.md [hunk 2]
```diff
@@ -145,6 +168,9 @@ nodeGroups:
       dedicated: tikv
     taints:
       dedicated: tikv:NoSchedule
+    iam:
```
Comment (csuzhangxc) [inline en/deploy-on-aws-eks.md:171]: `tikv-1d` 也需要？

**Review by csuzhangxc — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Comment by ran-huang** [comment]
Do these changes apply to release-1.5?

**Comment by csuzhangxc** [comment]
> Do these changes apply to release-1.5?

YES

**Pull request merged**